Improve commerce section (#149)
ezsystems/user-documentation: docs/shop_administration/manage_products.md

Product types, like Content Types, define the global properties of products and Fields a product consists of.
A product type also defines the attributes that all products of this type can have.

To create a product type:

1\. Go to **Product catalog** -> **Product Types** and click **Create**.

![Adding a product type](img/create_product_type.png)

2\. Fill in basic product information: name, identifier and description.

Each product has a product code that must be unique. It identifies the product in the system.
Product code can have up to 64 characters. It can contain only letters, numbers, underscores, and dashes.

> 3\. If needed, add additional Fields decriptinb the product.

Reviewer: ```suggestion
3\. If needed, add additional Fields describing the product.
```

4\. Add attributes.

## Attributes

Attributes describe a product's characteristics.
The customer can use them to filter and search for products.

Typical product attribute examples are: length, weight, color, format, and so on.

Attribute types define what type of information you can store in an attribute.
Available attribute types are:

- Checkbox
- Color - presented as a hex value
- Float - represents a number with fractions

> - Integer - represents a number without fractions

Reviewer: ```suggestion
- Integer - represents a number without fractions (a whole number)
```

- Measurement (range) - measurement with a given unit and minimum/maximum values selectable per product type
- Measurement (single) - measurement with a single value in given unit
- Selection - one of a list of customizable options

Each attribute belongs to an attribute group.
An example of an attribute group can be dimensions (which consists of length, width, height, and so on).

![Creating an attribute in a group](img/create_attribute.png)

When adding attributes to product types, you can add both individual attributes, and whole groups.

> You can also remove attributes from groups that you do not want to use for the given product type.

Reviewer: It's just a suggestion. What I think is missed, the instruction how we can remove attributes and that we have to drag and drop attributes.

## Product availability and stock

To control a product's availability and the available stock, in product or variant view go to **Availability** tab.
If a product can have variants, you must set availability for each variant separately.

![Product availability](img/product_availability_tab.png)

> Click **Create** to add availability to a product that doesn't have it, or **Edit** to modify existing availability settings.

Reviewer: ```suggestion
Click **Create** to add availability to a product that does not have it, or **Edit** to modify existing availability settings.
```

Reviewer: There are 2 Edit buttons. I think we should add that the button with edit icon must be clicked to edit the availability.

Author: Also added better screenshot which doesn't crop out the button
